build.gradle

```diff
@@ -3,7 +3,7 @@ buildscript {
         maven { url 'https://plugins.gradle.org/m2/' }
```

Comment:
we can update the version of the depot to 0.8.0.

Reply:
done